[ID1000322] implemente query the list of organization

Nixiangyu opened the pull request:
--story=1000322 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000322

Signed-off-by: Nixiangyu  <Nixiangyu@ofworld.com>

Commit:
[ID1000322] implemente query the list of organization

--story=1000322 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000322

Signed-off-by: Nixiangyu  <Nixiangyu@ofworld.com>

zhangbing16 commented on controllers/org/org.go:
```diff
@@ -0,0 +1,57 @@
+	logger.Infof("Got query organizations request")
+
+	req := &structs.QueryOrganizationsRequest{}
+	if err := c.BindJSON(req); err != nil {
```
应该使用BindQuery解析请求参数吧，这样能解析到请求的参数吗？

Nixiangyu replied:
好的，我修改一下

Commit:
[ID1000322] implemente query the list of organization

--story=1000322 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000322

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com

Commit:
[ID1000322] implemente query the list of organization

--story=1000322 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000322

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com
